torch/_inductor/graph.py

```diff
@@ -673,7 +673,7 @@ def compile_to_module(self):
         for name, value in self.constants.items():
             setattr(mod, name, value)
 
-        log.debug("Output code written to: %s", mod.__file__)
+        output_code_log.info("Output code written to: %s", mod.__file__)
```

Comment:
Hmm I'm thinking maybe we should just log it to both the `output_code` log and normal inductor log here. It will print twice in cases where `TORCH_LOGS=+inductor, output_code` but I think that's acceptable.